securedrop/requirements/update_python_dependencies

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Usage: ./update-python-dependencies
+# Run periodically to keep Python application, test, and development
+# requirements up-to-date.
+
+set -eC
+
+WORKING_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+declare -r WORKING_DIR
+declare -r VENV='review_env'
+declare -ra REQUIREMENTS_FILE_PREFIXES=('securedrop' 'test' 'develop')
+
+# This script should not be run with an active virtualenv. Calling deactivate
+# does not work reliably, so instead we quit with an error message.
+if [[ -n "${VIRTUAL_ENV}" ]]; then
+  printf '\nERROR: Please deactivate your virtualenv then re-run this script.\n'
+  exit 1
+fi
+
+# If pip and virtualenv are not available, try to install them with apt-get. If
+# apt-get is not available, quit with an error message.
+if ! command -v pip virtualenv > /dev/null; then
+  if command -v sudo apt-get > /dev/null; then
```

Comment:
You are using `command` incorrectly here. According to the [docs](https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html#Bash-Builtins), the usage is:

```
command [-pVv] command [arguments …]
```

That is, it only takes one command at a time. As a consequence, you cannot pass multiple arguments and rely on the return code to check for their existence. Here's an example Bash session:

```
vagrant@app-staging:~$ command -v echo
echo
vagrant@app-staging:~$ echo $?
0
vagrant@app-staging:~$ command -v notinstalled
vagrant@app-staging:~$ echo $?
1
vagrant@app-staging:~$ command -v echo notinstalled
echo
vagrant@app-staging:~$ echo $?
0
```